### PR TITLE
Display download links of profile results and dendrogram

### PR DESCRIPTION
**Pull request by yi-syong**

### CODE REVIEW

**Comment by chen1i6c04** [comment]
Looks good to me

File: frontend/src/components/Dendrogram_view.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	constructor(props) {
+		super(props);
+		this.state = {};
+		this.query_dengrogram = this.query_dengrogram.bind(this);
+	};
+
+	query_dengrogram(){
+
+		if(this.state.png_file == undefined){
+			fetch('api/profiling/dendrogram/'+window.batchid,{method:'GET'})
```
Comment (yuehhua) [inline frontend/src/components/Dendrogram_view.jsx]:
```suggestion
			fetch('api/profiling/dendrogram/' + window.batchid, {method: 'GET'})
```

File: frontend/src/components/Dendrogram_view.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+			.then(response => response.json())
+			.then(result => this.setState(state=>({png_file:result.png_file, pdf_file:result.pdf_file,
```
Comment (yuehhua) [inline frontend/src/components/Dendrogram_view.jsx]:
```suggestion
			.then(result => this.setState(state => ({png_file: result.png_file,
			                                         pdf_file: result.pdf_file,
```

File: frontend/src/components/Dendrogram_view.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+				svg_file:result.svg_file, emf_file:result.emf_file, newick_file:result.newick_file })));
```
Comment (yuehhua) [inline frontend/src/components/Dendrogram_view.jsx]:
```suggestion
				svg_file: result.svg_file,
				emf_file: result.emf_file,
				newick_file: result.newick_file })));
```

File: frontend/src/components/Dendrogram_view.jsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+			
+		}else {
+			clearInterval(this.interval);
+		}
+
+	}
+
+	componentDidMount(){
+		this.query_dengrogram();
+		this.interval = setInterval(this.query_dengrogram,30000);
```
Comment (yuehhua) [inline frontend/src/components/Dendrogram_view.jsx]:
```suggestion
		this.interval = setInterval(this.query_dengrogram, 30000);
```

File: frontend/src/components/Profile_view.jsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+	constructor(props) {
+		super(props);
+		this.state = {};
+		this.query_profile_result = this.query_profile_result.bind(this);
+	};
+
+	query_profile_result(){
+
+		if(this.state.profile_result_Url == undefined){
+			fetch('api/profiling/profile/'+window.batchid, {method:'GET'})
```
Comment (yuehhua) [inline frontend/src/components/Profile_view.jsx]:
```suggestion
			fetch('api/profiling/profile/' + window.batchid, {method: 'GET'})
```

File: frontend/src/components/Profile_view.jsx [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+			.then(response => response.json())
+			.then(result => this.setState(state=>({profile_result_Url:result.file})));
```
Comment (yuehhua) [inline frontend/src/components/Profile_view.jsx]:
```suggestion
			.then(result => this.setState(state => ({profile_result_Url: result.file})));
```

File: frontend/src/components/Upload.jsx [hunk 7]
```diff
@@ -35,23 +34,10 @@ class Main extends React.Component {
             autoProcessQueue: false,
             parallelUploads:200,
             init:function(){
-                this.on("sending",function(file,xhr,formData){
-                    formData.append("batch_id",window.batchid);
+                this.on("sending", function(file,xhr,formData){
```
Comment (yuehhua) [inline frontend/src/components/Upload.jsx]:
```suggestion
                this.on("sending", function(file, xhr, formData){
```

File: frontend/src/components/Upload.jsx [hunk 8]
```diff
@@ -35,23 +34,10 @@ class Main extends React.Component {
             autoProcessQueue: false,
             parallelUploads:200,
```
Comment (yuehhua) [inline frontend/src/components/Upload.jsx]:
```suggestion
            parallelUploads: 200,
```

File: frontend/src/components/Upload.jsx [hunk 9]
```diff
@@ -68,6 +54,18 @@ class Main extends React.Component {
         this.dropzone.processQueue();
     }
 
+    submit(){
+        var scheme = {};
+        scheme.occurrence = "95";
+        scheme.database = document.scheme.database.value;
+        scheme.id = window.batchid;
+        fetch('api/profiling/profiling/', {
+            method:'POST',
```
Comment (yuehhua) [inline frontend/src/components/Upload.jsx]:
```suggestion
            method: 'POST',
```

File: frontend/src/components/Upload.jsx [hunk 10]
```diff
@@ -68,6 +54,18 @@ class Main extends React.Component {
...
+            headers:new Headers({'content-type':'application/json'}),
```
Comment (yuehhua) [inline frontend/src/components/Upload.jsx]:
```suggestion
            headers: new Headers({'content-type': 'application/json'}),
```

File: frontend/src/components/Upload.jsx [hunk 11]
```diff
@@ -68,6 +54,18 @@ class Main extends React.Component {
...
+            body:JSON.stringify(scheme)
```
Comment (yuehhua) [inline frontend/src/components/Upload.jsx]:
```suggestion
            body: JSON.stringify(scheme)
```

**Comment by yuehhua** [comment]
Fixed.

**Pull request merged**